pipeline/runtime/runtestsV2.go

```diff
@@ -503,3 +505,18 @@ func collectTestReportsAndCg(ctx context.Context, log *logrus.Logger, r *api.Sta
 	}
 	return cgErr
 }
+
+func sanitizeTestGlobsV2(globStrings []string) []string {
+	var result = make([]string, 0)
+	for _, globString := range globStrings {
+		if globString != "" {
+			splitted := strings.Split(globString, ",")
```

Comment:
splits

Reply:
There was a case where we had a comma in "glob" which would break this. Let's make sure we cover such cases in input validation - maybe it was a Ruby pattern and not a glob.

Reply:
Lets treat it as a separate task since we will have to change the sanitization function of the current RunTestStep as well